lms/static/js/certificates/views/certificate_invalidation_view.js

```diff
@@ -0,0 +1,121 @@
+// Backbone Application View: CertificateInvalidationView
+/*global define, RequireJS */
+
+;(function(define) {
+    'use strict';
+    define(
+        ['jquery', 'underscore', 'gettext', 'backbone', 'js/certificates/models/certificate_invalidation'],
+
+        function($, _, gettext, Backbone, CertificateInvalidationModel) {
+            return Backbone.View.extend({
+                el: "#certificate-invalidation",
+                messages: "div.message",
+                events: {
+                    'click #invalidate-certificate': 'invalidateCertificate',
+                    'click .re-validate-certificate': 'reValidateCertificate'
+                },
+
+                initialize: function() {
+                    this.listenTo(this.collection, 'change add remove', this.render);
+                },
+
+                render: function() {
+                    var template = this.loadTemplate('certificate-invalidation');
+                    this.$el.html(template({certificate_invalidations: this.collection.models}));
+                },
+
+                loadTemplate: function(name) {
+                    var templateSelector = "#" + name + "-tpl",
+                    templateText = $(templateSelector).text();
+                    return _.template(templateText);
+                },
+
+                invalidateCertificate: function() {
+                    var user = this.$("#certificate-invalidation-user").val();
+                    var notes = this.$("#certificate-invalidation-notes").val();
+
+                    var certificate_invalidation = new CertificateInvalidationModel({
+                        url: this.collection.url,
+                        user: user,
+                        notes: notes
+                    });
+
+                    if (this.collection.findWhere({user: user})) {
+                        this.showMessage(
+                            gettext("Certificate of ") + user +
+                            gettext(" has already been invalidated. Please check your spelling and retry."
```

Comment:
You can't concatenate translated strings like this.  The entire sentence has to be a single string for translation. cc: @mattdrayer